.github/workflows/release.yml

```diff
@@ -45,6 +45,12 @@ jobs:
   
   build:
     needs: check
+    outputs:
+      linux-x64-sha: ${{ steps.sha.outputs.linux-x64-sha256 }}
```

Comment:
curious whether `${{ matrix.runtime }}-sha: ${{ steps.compute-sha.outputs.sha }}` would work?

Reply:
ok to be explicit too... looks like there is another place below where we have to repeat the runtimes, so whatevs :)

Reply:
other way around repeating platforms in multiple places might be to upload as part of artifact